CMakeLists.txt

```diff
@@ -104,6 +104,12 @@ if(APPLE)
   endif()
 endif()
 
+if(DEFINED VCPKG_ROOT AND DEFINED VCPKG_TARGET_TRIPLET)
+  if(NOT EXISTS "${VCPKG_ROOT}/installed/${VCPKG_TARGET_TRIPLET}")
+    message(FATAL_ERROR "VCPKG_TARGET_TRIPLET dir not found: ${VCPKG_ROOT}/installed/${VCPKG_TARGET_TRIPLET}")
```

Comment:
this is still very opaque for people that don't know what this means.

Reply:
Ok, I have added more context.
Is is OK now?